Sources/BokenEngine/Classes/StoryParser.swift

```diff
@@ -20,14 +22,17 @@ extension StoryParserError: LocalizedError {
         case .decodingError:
             return NSLocalizedString("A decoding error ocurred when decoding the story description.",
                                      comment: "storyParserErrorDecodingError")
+        case .readError:
+            return NSLocalizedString("Could not access mocked Story Description file.",
+                                     comment: "storyParserErrorReadingMockedStoryDescription")
         }
     }
 }
 
 class StoryParser {
```

Comment:
@yeraydavidrodriguez I think I would split this class into a least 2 classes: StoryParser and StoryFileLoader but since it's not related to the PR we can leave it that way and discuss it later.

Reply:
Yes, that would be a nice refactor. Will add it as an issue.